site/en/tutorials/python_quickstart.ipynb

```diff
@@ -1034,6 +1034,66 @@
         "  display(to_markdown(f'**{message.role}**: {message.parts[0].text}'))"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AEgVOYu0pAr4"
+      },
+      "source": [
+        "## Counting number of Tokens\n",
+        "\n",
+        "All language models have a context window which is often mentioned in terms of **number of tokens**. In order to utilize the full potential of these language models, determining the number of tokens involved in `ChatSessions` or `prompts` is crucial.\n",
+        "\n",
+        "Gemini enables you to count the number of tokens involved in any `glm.Content` object. In the simplest case, you can pass a query string to the `GenerativeModel.count_tokens` method:"
```

Comment:
The "to utilize the full potential" of LLMs, "determining the number of tokens" may be a bit subjective, without providing a clear explanation why it's "crucial". Maybe it's an open area of research.

How about:

```suggestion
        "Large language models have a context window, and the context length is often measured in terms of the **number of tokens**. With the Gemini API, you can determine the number of tokens per any `glm.Content` object. In the simplest case, you can pass a query string to the `GenerativeModel.count_tokens` method as follows:"
```

Reply:
@MarkDaoust cc

Reply:
@pcoet cc

Reply:
@8bitmp3 makes a good suggestion. Feel free to make this change and ignore mine below.

Reply:
Done with all of the suggested changes. Thank you